.pre-commit-config.yaml

```diff
@@ -7,17 +7,17 @@ repos:
     hooks:
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
+-   repo: https://github.com/crate-ci/typos
+    rev: v1.23.1
+    hooks:
+      - id: typos
 -   repo: local
     hooks:
     -   id: rustfmt
         name: rustfmt
         entry: rustfmt --edition 2021
         language: system
         types: [rust]
-    -   id: typos
-        name: typos
-        entry: typos -w
-        language: system
```

Comment:
Because previously it's trying to fix typos in png 😟

Reply:
But you also added an item of `repo: https://github.com/crate-ci/typos`. What's the difference?

Reply:
official https://github.com/crate-ci/typos/blob/master/docs/pre-commit.md